.github/workflows/ci.yml

```diff
@@ -15,8 +15,8 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04]
-        python-version: ['3.8']
-        toxenv: [docs, quality, django32, django42]
+        python-version: ['3.8', '3.11', '3.12']
```

Comment:
@awais786: In order to save build time and resources, is it ok to only add 3.12 to start, and we can add 3.11 just to those that start failing? That's what some of us had in mind anyway. Let me know your thoughts.

Reply:
That said, I guess it makes sense to experiment on one repo, because now we see that 3.11 is passing and that we just have a problem linting in 3.12. Carry on. We'll see what you learn. Thanks.

Reply:
Got it, I will fix all the prs with python312 only.